po/uk.po

```diff
@@ -608,155 +608,156 @@ msgstr "використовувати режим накладення"
 
 #: builtin/clone.c:48
 msgid "git clone [<options>] [--] <repo> [<dir>]"
-msgstr ""
+msgstr "git clone [<опції>] [--] <репозиторій> [<директорія>]"
 
 #: builtin/clone.c:102
 msgid "don't clone shallow repository"
-msgstr ""
+msgstr "не клонувати неглибокий репозиторій"
 
 #: builtin/clone.c:104
 msgid "don't create a checkout"
-msgstr ""
+msgstr "не переходити до гілки"
 
 #: builtin/clone.c:105 builtin/clone.c:107
 msgid "create a bare repository"
-msgstr ""
+msgstr "створити порожній репозиторій"
 
 #: builtin/clone.c:109
 msgid "create a mirror repository (implies bare)"
-msgstr ""
+msgstr "створити дзеркальний репозиторій (включає опцію bare)"
```

Comment:
```suggestion
msgstr "створити дзеркальний репозиторій (мається на увазі порожній репозиторій)"
```

Reply:
I'm not sure if the resulting repository would be empty here. My understanding is that it just uses --bare.